packages/ui/src/components/verifyCodeInput/verifyCodeInput.styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { Button as AntButton } from 'antd';
+import * as colors from '../../colors';
+import * as mixins from '../../mixins';
+
+export const SendEmailButton = styled(AntButton)`
+  height: 22px !important;
+  line-height: 22px !important;
+  padding: 0 4px !important;
```

Comment:
We have already overwritten the AntD buttons for different sizes and use cases like it's mentioned in the UI doc.
Please refer to https://tidb-community-ui-git-cpt-108-form-validation-tidb-community.vercel.app/?path=/story/ant-design-button--primary

Should we align with the same sizes but not introducing any new sizes?

<img width="1213" alt="Screen Shot 2021-04-30 at 7 10 45 PM" src="https://user-images.githubusercontent.com/123074/116660929-ce61fd80-a9e7-11eb-915f-4dcb48543ee6.png">

Reply:
It is a special button that placed into a `Input` element, and it's default size will break it's parent style. We could discuss this at final.